src/Explorer/Controls/TreeComponent/Styles.ts

```diff
@@ -10,6 +10,7 @@ const actionButtonBackground = "--cosmos-Tree--actionButtonBackground" as const;
 export const useTreeStyles = makeStyles({
   treeContainer: {
     height: "100%",
+    maxHeight: "calc(100vh - 100px)", // Use viewport-relative height
```

Comment:
Do you know where the 100px come from? I'd rather not add this is we don't need to.